test/mongo/connection_test.exs

```diff
@@ -94,10 +95,10 @@ defmodule Mongo.ConnectionTest do
             username: "mongodb_admin_user", password: "wrong",
             backoff_type: :stop, auth_source: "admin_test"]
 
-    capture_log fn ->
-      assert {:ok, pid} = Mongo.start_link(opts)
-      assert_receive {:EXIT, ^pid, {%Mongo.Error{code: 18}, _}}
-    end
+    assert capture_log(fn ->
+      {:ok, pid} = Mongo.start_link(opts)
+      assert_receive {:EXIT, ^pid, :killed}, 5000
+    end) =~ "(Mongo.Error) auth failed for user mongodb_admin_user"
```

Comment:
What was the purpose of changing the asserts?

Reply:
after I updated DBConnection, the old way of capturing log is no longer useful, so I use `ExUnit.CaptureLog` to make sure it did return some error message